ydb/aio/query/session.py

```diff
@@ -58,7 +58,9 @@ async def _check_session_status_loop(self) -> None:
                     self._state.reset()
                     self._state._change_state(QuerySessionStateEnum.CLOSED)
         except Exception:
-            pass
+            if not self._state._already_in(QuerySessionStateEnum.CLOSED):
+                self._state.reset()
```

Comment:
1. Why do you need reset session state before set closed _state._state?
2. When the session will be really closed (call `DeleteSession` and detach from attached stream?